modules/test/playwright/tests/frontend-data-set-views-web/settings.spec.ts

```diff
@@ -222,12 +187,442 @@ test.describe('Data Set Settings', () => {
 					settingsPage.defaultVisualizationModeLabel
 				).toContainText('List');
 			});
-
-			await test.step('Clean data', async () => {
-				await dataSetManagerApiHelpers.deleteDataSet({
-					erc: customDataSetERC,
-				});
-			});
 		});
 	});
 });
+
+export const fragmentTest = mergeTests(
+	apiHelpersTest,
+	dataSetManagerApiHelpersTest,
+	featureFlagsTest({
+		'LPD-10735': true,
+		'LPS-164563': true,
+		'LPS-178052': true,
+	}),
+	fdsFragmentPageTest,
+	isolatedSiteTest
+);
+
+fragmentTest.describe('Data Set Default Visualization Mode in fragment', () => {
+	fragmentTest(
+		'Default Visualization Mode is not configured (no field for any section). Use Table as default',
+		async ({
+			apiHelpers,
+			dataSetManagerApiHelpers,
+			fdsFragmentPage,
+			page,
+			site,
+		}) => {
+			await test.step('Create random data (fields) for other Data Sets', async () => {
+				await dataSetManagerApiHelpers.createDataSetViewFields({});
+			});
+
+			const layout = await fragmentTest.step(
+				'Create a new page',
+				async () => {
+					const pageLayout =
+						await apiHelpers.headlessDelivery.createSitePage(
+							site.id,
+							getRandomString()
+						);
+
+					return pageLayout;
+				}
+			);
+
```

Comment:
I've tried to create a page including the Data Set fragment using:

```
const fdsFragmentDefinition = getFragmentDefinition(
	getRandomString(),
	'com.liferay.frontend.data.set.views.web.internal.fragment.renderer.FDSViewFragmentRenderer'
);

const layout = await fragmentTest.step('Create a new page', async () => {
	const pageLayout = await apiHelpers.headlessDelivery.createSitePage(
		site.id,
		getRandomString(),

		// getPageDefinition([fdsFragmentDefinition])

	);

	return pageLayout;
});
```

However, it is failing due to some missing js modules. Need to investigate a bit more